bundles/org.eclipse.passage.lbc.server.launcher/META-INF/MANIFEST.MF

```diff
@@ -8,3 +8,8 @@ Bundle-Vendor: %Bundle-Vendor
 Bundle-Copyright: %Bundle-Copyright
 Bundle-RequiredExecutionEnvironment: JavaSE-1.8
 Bundle-ActivationPolicy: lazy
+Service-Component: OSGI-INF/org.eclipse.passage.lbc.server.launcher.ApplicationLauncher.xml
+Require-Bundle: org.eclipse.osgi.services;bundle-version="0.0.0",
+ org.eclipse.passage.lbc.api;bundle-version="0.0.0",
+ org.eclipse.passage.lbc.jetty;bundle-version="0.0.0"
+Export-Package: org.eclipse.passage.lbc.server.launcher
```

Comment:
This package should be marked as internal, or better renamed to have "internal" in its name